Client.Examples/Program.cs

```diff
@@ -113,7 +113,7 @@ private static void HandleJob(IJobClient jobClient, IJob job)
             if (jobKey % 3 == 0)
             {
                 jobClient.NewCompleteJobCommand(jobKey)
-                    .Variables("{\"foo\":2}")
+                    .Variables("{\"payment-service\":2}")
```

Comment:
This will probably not work due to the output mappings :)

Reply:
I updated the code and commit message